permutableString.js

```diff
@@ -0,0 +1,38 @@
+// https://dmitripavlutin.com/how-to-compare-objects-in-javascript/
+//Link above - compare objects, 3 subheader - don't clear 'Object.keys'...
+//Works when both of strings are the same length.
+function permutable(s1, s2) {
+    let obj1 = {}, obj2 = {};
```

Comment:
You can count symbols here. It is better to declare them `const` because they are not changed. 
```
const obj1 = countSymbols(s1);
const obj2 = countSymbols(s2);
```